hazelcast/src/main/java/com/hazelcast/spi/InvocationBuilder.java

```diff
@@ -240,5 +241,10 @@ protected ExecutionCallback getTargetExecutionCallback() {
         return executionCallback;
     }
 
+    public InvocationBuilder setDoneCallback(Runnable doneCallback) {
```

Comment:
I already had that description on Invocation Future, but then it was lost in a refactoring step. It is not re-introduced on this method, plus a reference to it from `Invocation.taskDoneCallback`.